data/tests/apg/checkbox-two-state.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "APG 1.1 Checkbox Example (Two State)",
   "description": "This example implements the Checkbox Design Pattern for a two state checkbox using div elements.",
-  "html_file": "https://w3c.github.io/aria-practices/examples/checkbox/checkbox-1/checkbox-1.html",
+  "html_file": "https://www.w3.org/TR/wai-aria-practices-1.1/examples/checkbox/checkbox-1/checkbox-1.html",
```

Comment:
might the versionless URL be used instead, so that this doesn't become out of date again once 1.2 goes to published note state? https://w3c.github.io/aria-practices/examples/checkbox/checkbox.html

Reply:
Good call @scottaohara , I agree the versionless URL won't affect test outcomes in this case.